src/ShareCreated.php

```diff
@@ -55,4 +55,37 @@ public function setRole(SharingRole $role): bool
         $this->apiPermission = $apiPermission;
         return true;
     }
+
+    /**
+     * @throws UnauthorizedException
+     * @throws ForbiddenException
+     * @throws InvalidResponseException
+     * @throws HttpException
+     * @throws BadRequestException
+     * @throws NotFoundException
+     */
+    public function getReceiver(): User|Group
+    {
+        $ocis = new Ocis($this->serviceUrl, $this->accessToken, $this->connectionConfig);
```

Comment:
is this a legitimate approach?
I wanted to have `getUserById` and `getGroupById` in the `Ocis`class, but also feels strange to create an `Ocis` object here.
Alternatively I could have `getUserById` and `getGroupById` in some other helper class.

Reply:
yes, but which one and how to keep it so, that its easy to use for the developers?